### DISCUSSION TITLE
Licensing for Self Hosted Product Documentation

### DISCUSSION THREAD

**Discussion by andrewkcarter** [outline/outline#7992] [Self Hosting] [answered]
Our use case is that we would like to self-host outline Community Edition to serve product documentation for our commercial platform. Our employees will be editors, and our customers will have read-only viewer access to Outline. We will not form teams of our end users or allow them to edit or author content within Outline.

From your perspective, is this setup in line with and honoring the additional use grant?

> Additional Use Grant: You may make use of the Licensed Work, provided that
                      you may not use the Licensed Work for a Document
                      Service.
>
> A “Document Service” is a commercial offering that
                      allows third parties (other than your employees and
                      contractors) to access the functionality of the
                      Licensed Work by creating teams and documents
                      controlled by such third parties.

**Comment by tommoor** [comment, marked as answer]
Yes - this usecase is within the terms of the license

**Reply by andrewkcarter** [reply]
I appreciate the quick reply.
